### PR TITLE
Provide defaults in form

### PR DESCRIPTION
**Pull request by Netznarkose**

### CODE REVIEW

File: app/models/chart.rb [hunk 1]
```diff
@@ -1,15 +1,8 @@
 class Chart < ActiveRecord::Base
   has_many :data
-  accepts_nested_attributes_for :data, reject_if: :all_blank, allow_destroy: true
```
Comment (eikes) [inline app/models/chart.rb]: why not reject_if and why not allow_destroy ?

File: app/views/charts/_form.html.erb [hunk 2]
```diff
@@ -16,21 +16,21 @@
       <% end %>
       <div class="well">
         <%= f.input(:title) %>
-        <%= f.input(:background_color, input_html: { class: :minicolors } ) %>
+        <%= f.input(:background_color, input_html: { class: :minicolors, value: f.object.background_color || '#ffffff' } ) %>
```
Comment (eikes) [inline app/views/charts/_form.html.erb]: why is this not in the initializer anymore?

File: app/models/chart.rb (hunk 1 above)
Comment (Netznarkose) [inline app/models/chart.rb]: there is individual validation in datum.rb going on.
if we set `reject_if` and `allow_destroy` our error-messages become less descriptive.

Comment (eikes) [inline app/models/chart.rb]: but without allow_destroy you cannot remove a datum from a chart

Comment (Netznarkose) [inline app/models/chart.rb]: indeed, I didn't had this in mind! I'll put it the `allow_destroy` back in.

**Pull request closed without merging**